src/CSnakes.Tests/GeneratedSignatureTests.cs

```diff
@@ -226,4 +229,17 @@ public void TestUnionReturnType(string returnType)
         var module1 = ModuleReflection.MethodsFromFunctionDefinitions(functions1).ToImmutableArray();
         Assert.Single(module1);
     }
+
+    [Theory]
+    [InlineData("def foo(  # csharp: ignore\r\n        bar,\r\n        baz,\r\n    ):\r\n    pass\r\n")]
+    [InlineData("def hello(): # csharp: ignore ...\n")]
+    [InlineData("def foo():  # type: ignore # csharp: ignore\r\n    pass\r\n\r\ndef bar ():  # csharp: ignore # type: ignore\r\n    pass")]
```

Comment:
What about separating this into two cases so if one fails, it's easier to identify which?